velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -501,3 +502,46 @@ TEST_F(PlanNodeToStringTest, tableScan) {
         plan->toString(true, false));
   }
 }
+
+TEST_F(PlanNodeToStringTest, window) {
+  std::vector<exec::FunctionSignaturePtr> signatures{
+      exec::FunctionSignatureBuilder()
+          .argumentType("BIGINT")
+          .returnType("BIGINT")
+          .build(),
+  };
+  exec::registerWindowFunction("window1", std::move(signatures), nullptr);
+
+  PlanBuilder::WindowFrame frame = {
+      core::WindowNode::WindowType::kRange,
+      core::WindowNode::BoundType::kPreceding,
+      "10",
+      core::WindowNode::BoundType::kFollowing,
+      "b"};
+  auto plan =
+      PlanBuilder()
+          .tableScan(ROW({"a", "b", "c"}, {VARCHAR(), BIGINT(), BIGINT()}))
+          .window({"a"}, {"b"}, {"window1(c) AS d"}, {frame})
+          .planNode();
+  ASSERT_EQ(
+      "-- Window[partition by: [a] order by: [b ASC NULLS LAST] "
+      "d := window1(ROW[\"c\"]) RANGE 10 PRECEDING ROW[\"b\"] FOLLOWING] "
```

Comment:
Is `ROW[\"c\"]` and `ROW[\"b\"]` accurate? The plan just refers by `c`, `b`.

Reply:
ROW[\"c\"] in the function invocation cannot be avoided on account of the CallTYpedExprPtr toString() implementation. However, I've changed to avoid that usage in the window frame now.